### PR TITLE
[RDAPI-14100] Adding performance recommendation for API Gateway configuration

### PR DESCRIPTION
**Pull request by dsantosaxway**
## Describe the changes

Adding performance recommendation for API Gateway configuration

## Deploy preview link

http://pr-2534.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/cass_admin/cassandra_architecture/index.html#api-gateway-configuration

## Checklist for contributors

Before submitting this PR, please make sure:

_This is simply a reminder of what we are going to look for before merging your change._

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [X] You have verified that all status checks have passed

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 90277fa |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2534.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/cass_admin/cassandra_architecture.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ API Gateway acts as a client of the Cassandra cluster as follows:
 * Strong consistency
   : Cassandra read and write consistency levels are both set to `QUORUM`. This, along with the replication factor of `3`, enables full availability in the event of one node loss.
 
+API Gateway for performance should not be located on the same server as Cassandra.
```
Comment (kmercierwhite) [inline content/en/docs/cass_admin/cassandra_architecture.md:131]: Reword to:

Cassandra and API Gateway must avoid taking up the same system resources. For this reason, Cassandra should be installed on a different server than the API Gateway.

Comment (dsantosaxway) [reply content/en/docs/cass_admin/cassandra_architecture.md]: Done!

**Review by kmercierwhite — APPROVED** [review]
Cassandra should be installed in seperate

**Comment by kmercierwhite** [comment]
> Cassandra should be installed in seperate

Ignore this, added as a mistake while I was formulating a comment for my review.

**Pull request merged**